add-domain

```diff
@@ -768,3 +768,5 @@ soclaiemx.xyz
 wiflix.xyz
 wiflix.yachts
 coronavirus.zone
+gіthսᖯ.com
+ɡithub.com
```

Comment:
First, do not add the same record twice.
Second, I do admit that GitHub, GitLab and BitBucket and a verity of other git host are full of malicious, phishing and tracking script that are being hosted with the dull blessing from those services. 

However, we need the full url to the script (repo)

You'll properly also like to test out access from `raw.githubusercontent.com` and `github.io`

----------------

## Search result from External Hosts-Sources
@mypdns's External Hosts-Sources can be found [here](https://github.com/external-sources/hosts-sources?tab=readme-ov-file#external-sources)

```css
data/ShadowWhispererTracking.csv:collector-cdn.github.com
data/ShadowWhispererTracking.csv:collector.github.com
data/blocklistproject.csv:collector-cdn.github.com
data/blocklistproject.csv:pivotal.github.com
data/hosts-file.ad_servers.csv:collector-cdn.github.com
data/hosts-file.ad_servers.csv:pivotal.github.com
data/migueldemouraAds-tracking.csv:collector.github.com
data/quidsUpMixed.csv:collector.github.com
```

#### Sorted result

```css
collector-cdn.github.com
collector.github.com
pivotal.github.com
```

### Search result from easylist

```css
easylist/easylist_allowlist.txt:@@||github.com^$generichide
easylist/easylist_general_block.txt:/adsmanager/*$domain=~adsmanager.facebook.com|~github.com
easylist/easylist_general_block.txt:/adverts.$~script,domain=~0xacab.org|~adverts.ie|~adverts.org.ua|~github.com|~gitlab.com
easylist/easylist_general_block.txt:/reklam.$domain=~github.com|~reklam.com.tr
easylist/easylist_general_block.txt:/reklam/*$domain=~cloudflare.com|~github.com|~reklam.com.tr
easylist/easylist_general_block.txt:/reklama/*$domain=~github.com
easylist_cookie/easylist_cookie_specific_hide.txt:github.com###cookie-consent-banner
easylist_cookie/easylist_cookie_specific_hide.txt:apgbio.com,cheddar.com,co-operative.coop,couchsurfing.com,cyscale.com,diagnosticquestions.com,fluidra.ba,fullscreen.com,haymarketbooks.org,heimat-info.de,human-facts.com,incyte.jp,jumia.ci,jumia.co.ke,jumia.com.dz,jumia.com.eg,jumia.com.gh,jumia.com.ng,jumia.com.tn,jumia.dz,jumia.ma,jumia.sn,jumia.ug,kaleido.ai,modrinth.com,oceanmata.com,peer5.com,pluralsight.com,securitylab.github.com,subaru.be,trainsplit.com,usatoday.com,versobooks.com,zando.co.za##.banner
easylist_cookie/easylist_cookie_specific_hide.txt:support.github.com###consent-banner
easylist_cookie/easylist_cookie_specific_hide.txt:github.com##.hx_cookie-banner
easyprivacy/easyprivacy_allowlist.txt:@@||github.com/gorhill/uBlock/*/src/web_accessible_resources/fingerprint2.js$~third-party
easyprivacy/easyprivacy_general.txt:/event-tracking/*$domain=~github.com
easyprivacy/easyprivacy_general.txt:/matomo.js$domain=~github.com
easyprivacy/easyprivacy_general.txt:/matomo/*$domain=~github.com|~hub.docker.com|~matomo.org|~wordpress.org
easyprivacy/easyprivacy_general.txt:/piwik-$domain=~github.com|~matomo.org|~piwik.org|~piwik.pro|~piwikpro.de
easyprivacy/easyprivacy_general.txt:/piwik/*$domain=~github.com|~matomo.org|~piwik.org|~piwik.pro
easyprivacy/easyprivacy_specific.txt:||collector.github.com^
easyprivacy/easyprivacy_specific.txt:||docs.github.com/events
easyprivacy/easyprivacy_specific.txt:||github.com/_private/browser/stats
easyprivacy/easyprivacy_specific.txt:||support.github.com/_event
easyprivacy/easyprivacy_specific.txt:||support.github.com/_stats
easyprivacy/easyprivacy_specific.txt:||support.github.com/session/event
easyprivacy/easyprivacy_specific.txt:||support.github.com/session/user
easyprivacy/easyprivacy_thirdparty.txt:! https://gist.github.com/tinogomes/c425aa2a56d289f16a1f4fcb8a65ea65
fanboy-addon/fanboy_annoyance_specific_hide.txt:github.com##.signup-prompt
fanboy-addon/fanboy_annoyance_specific_hide.txt:github.com##.js-notice-dismiss[data-analytics-visible]
fanboy-addon/fanboy_annoyance_specific_hide.txt:docs.github.com##[aria-label="Scroll to top"]
fanboy-addon/fanboy_annoyance_specific_uBO.txt:github.com##button[data-testid="copilot-popover-button"]
fanboy-addon/fanboy_notifications_general_block.txt:-push-notification.$domain=~github.com
fanboy-addon/fanboy_notifications_general_block.txt:-push-notification/$domain=~github.com
fanboy-addon/fanboy_social_allowlist.txt:@@||github.com^$generichide
```

## Search in [Matrix](https://www.mypdns.org/)


Search results from [Matrix](https://www.mypdns.org) blacklist project

```css
source/tracking/domains.csv:alive.github.com
source/tracking/wildcard.csv:github.com
```


Found these RPZ records from My Privacy DNS
```sql
id      domain records  type    content
21295921        alive.github.com.tracking.mypdns.cloud  CNAME   .
21295923        collector.github.com.tracking.mypdns.cloud      CNAME   .
21295922        live.github.com.tracking.mypdns.cloud   CNAME   .
24844905        *.github.com.scamming.mypdns.cloud      CNAME   .
24844904        github.com.scamming.mypdns.cloud        CNAME   .
24818836        alive.github.com.adware.mypdns.cloud    CNAME   .
```


Any known matrix issues?

\+ We did not find any existing issues for the domain in Matrix

+++++++++++++++++++++++++++++++++++++++
\+ Thanks to [My Privacy DNS](https://www.mypdns.org/) for this knowledge +
+++++++++++++++++++++++++++++++++++++++

----------------

Reply:
> First, do not add the same record twice.
They look similar, but they are not the same.

> Second, I do admit that GitHub, GitLab and BitBucket and a verity of other git host are full of malicious, phishing and tracking script that are being hosted with the dull blessing from those services.
I totally agree with you.